### PR TITLE
chore: upload deb package to artifacts only

### PR DESCRIPTION
**Pull request by soulbird**
Upload deb package to artifacts only, for circleci cant't upload debian package file to cos.

### CODE REVIEW

File: .github/workflows/publish-deb.yml [hunk 1]
```diff
@@ -26,7 +23,6 @@ jobs:
       - name: Init basic publish env
         run: |
           sudo apt-get update
-          sudo apt install -y gpg ca-certificates
```
Comment (tzssangglass) [inline .github/workflows/publish-deb.yml:29]: remove `gpg` means that we need to sign the generated deb artifacts manually after we get them?

**Comment by tzssangglass** [comment]
have a CI running successfully in your own repository?

File: .github/workflows/publish-deb.yml (hunk 1 above)
Comment (soulbird) [reply .github/workflows/publish-deb.yml:29]: Just upload to CI Action, no signing required. The debian repo currently needs to be built locally and uploaded to cos.

**Comment by soulbird** [comment]
> have a CI running successfully in your own repository?

https://github.com/api7/apisix-build-tools/actions/runs/2655676599
https://app.circleci.com/pipelines/github/api7/apisix-build-tools/154/workflows/12db3bf2-e065-441b-b00c-c9c9ea2104d5/jobs/221/artifacts

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**